.github/workflows/helm-release.yaml

```diff
@@ -76,36 +77,16 @@ jobs:
           name: ${{ steps.package.outputs.filename }}
           path: .cr-release-packages/${{ steps.package.outputs.filename }}
           retention-days: ${{ github.event_name == 'pull_request' && 1 || 30 }}
-
-  release:
-    name: Release
-    runs-on: ubuntu-latest
-    needs: package
-    environment: charts
-    # https://github.com/actions/runner/issues/1483
-    if: github.event.inputs.release == 'true'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Install chart-releaser
-        run:
-          curl -L "https://github.com/helm/chart-releaser/releases/download/v${CR_VERSION}/chart-releaser_${CR_VERSION}_linux_amd64.tar.gz"
-          | sudo tar xvz --directory /usr/bin cr
-      - name: Download packaged chart
-        uses: actions/download-artifact@v3
-        with:
-          name: ${{ needs.package.outputs.filename }}
-          path: .cr-release-packages
       - name: Generate GitHub app token
         id: github-app-token
+        if: github.event.inputs.release == 'true'
```

Comment:
Hmm I kind of liked the job level separation and just having one top level place where things are skipped, seems less error prone

Reply:
I don't mind either way. It was just a lot of duplication (especially once you add the cache as well)